frontend/src/lib/components/FilterPropertyLink.js

```diff
@@ -4,11 +4,20 @@ import PropTypes from 'prop-types'
 
 import { Property } from 'lib/components/Property'
 import { Link } from 'lib/components/Link'
+import { parseProperties } from 'lib/components/PropertyFilters/propertyFilterLogic'
 
 export function FilterPropertyLink({ property, value, filters, onClick }) {
+    const cleanedProperties = Array.isArray(filters.properties)
+        ? filters.properties
+        : parseProperties(filters.properties)
+
+    const properties = cleanedProperties.find(p => p.key === property && p.value === value && !p.operator)
```

Comment:
I spent some time looking at this trying to work out what this does and i'm still not 100% sure. Maybe worth adding a comment explaining?

Reply:
Like this? https://github.com/PostHog/posthog/pull/769/commits/f2206806a52623f46cd7116d5370dd09c948d39c

Reply:
Yep perf